metecho/api/models.py

```diff
@@ -451,6 +453,60 @@ def get_head(self):
 
     # end CreatePrMixin configuration
 
+    def create_gh_branch(self):
+        if not self.id:  # i.e. "newly created"
+            # We cannot get an app user with a repo_id, sadly; the API
+            # doesn't support it. So we have to use the repo_owner and
+            # repo_name.
```

Comment:
@wlonk Can you explain this a bit more? Why do we need a different `repository` object here than we use in e.g. `jobs._create_branches_on_github`? What is the "app user" and why do we need it here in a different way?